src/Xamarin.Android.Build.Tasks/Tasks/ResolveAssemblies.cs

```diff
@@ -90,7 +90,9 @@ void Execute (MetadataResolver resolver)
 				foreach (var assembly in Assemblies) {
 					var assembly_path = Path.GetDirectoryName (assembly.ItemSpec);
 					resolver.AddSearchDirectory (assembly_path);
-
+					if (string.Compare (assembly.GetMetadata ("AndroidDontIncludeInPackage"), bool.TrueString, StringComparison.OrdinalIgnoreCase) == 0) {
```

Comment:
This might be better to use the Skip prefix like the others: SkipAddToPackage or something

Reply:
I renamed all the metadata to be `AndroidSkip` prefixed. This is inline with our other properties.